src/Makefile

```diff
@@ -639,3 +639,25 @@ uninstall:
 	$(call MAYBE_UNINSTALL_REDIS_SYMLINK,$(INSTALL_BIN),$(ENGINE_CHECK_RDB_NAME))
 	$(call MAYBE_UNINSTALL_REDIS_SYMLINK,$(INSTALL_BIN),$(ENGINE_CHECK_AOF_NAME))
 	$(call MAYBE_UNINSTALL_REDIS_SYMLINK,$(INSTALL_BIN),$(ENGINE_SENTINEL_NAME))
+
+format:
+	@if ! command -v clang-format >/dev/null 2>&1; then \
+		echo "Error: clang-format is required to run this command."; \
+		exit 1; \
+	fi
+	@CLANG_FORMAT_VERSION=$$(clang-format --version | awk '{print $$3}' | cut -d. -f1); \
+	if [ $$CLANG_FORMAT_VERSION -lt 18 ]; then \
```

Comment:
I  think we will have to stick to clang 18 for now. I have run into some behavior differences between 18 and 19.